exec/jdbc/src/test/java/org/apache/drill/jdbc/PreparedStatementTest.java

```diff
@@ -237,6 +245,127 @@ public String toString() {
     }
   }
 
+  /**
+   * Test for reading of default query timeout
+   */
+  @Test
+  public void testDefaultGetQueryTimeout() throws SQLException {
+    PreparedStatement stmt = connection.prepareStatement(SYS_VERSION_SQL);
+    int timeoutValue = stmt.getQueryTimeout();
+    assert( 0 == timeoutValue );
+  }
+
+  /**
+   * Test Invalid parameter by giving negative timeout
+   */
+  @Test ( expected = InvalidParameterSqlException.class )
+  public void testInvalidSetQueryTimeout() throws SQLException {
+    PreparedStatement stmt = connection.prepareStatement(SYS_VERSION_SQL);
+    //Setting negative value
+    int valueToSet = -10;
+    if (0L == valueToSet) {
+      valueToSet--;
+    }
+    try {
+      stmt.setQueryTimeout(valueToSet);
+    } catch ( final Exception e) {
+      // TODO: handle exception
+      assertThat( e.getMessage(), containsString( "illegal timeout value") );
+      //Converting this to match expected Exception
+      throw new InvalidParameterSqlException(e.getMessage());
```

Comment:
there's no point I guess? you already verified the exception...

Reply:
Wanted to make sure that the unit test also reports the correct exception. 
This only rewraps the thrown SQLException to an InvalidParameterSqlException for JUnit to confirm.

Reply:
but you did since you catch the exception and do a check on the message. Rewrapping it so that the test framework can check the new type has no value.

Reply:
Yep. Agree. Was trying to make use of the large number of `###SqlException`s defined within the Drill JDBC package. Will fix this. +1